kitchen-tests/Gemfile.lock

```diff
@@ -47,15 +47,15 @@ GEM
     celluloid-io (0.16.2)
       celluloid (>= 0.16.0)
       nio4r (>= 1.1.0)
-    chef-config (12.19.36)
+    chef-config (12.20.3)
```

Comment:
that was necessary before 13 was released, it can probably be fixed now, but its not relevant to this mess.